.changeset/three-months-lie.md

```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/firestore": patch
+---
+
+Fix a bug that prevented aggregations from being run when muli-tab was enabled.
```

Comment:
nit: I'd just recommend saying "sometimes" in this change log entry. The way it is worded IMO incorrectly implies that count queries didn't work at all in multi-tab mode.

Reply:
Done